uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseGridWidget.java

```diff
@@ -244,6 +248,12 @@ public double getHeight() {
         return height;
     }
 
+    private double getHeight(final BaseGridRendererHelper.RenderingInformation renderingInformation) {
+        double height = renderer.getHeaderHeight();
+        height = height + renderingInformation.getAllRowHeights().stream().reduce(0d, Double::sum);
```

Comment:
@manstis 
pls replace with for-loop (or parallel stream) for performance

Reply:
Done.